app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,128 @@
-// Code your JavaScript / jQuery solution here
+var turn = 0;
+var currentGame = 0;
+
+var  winCombinations = [
```

Comment:
Since these will never change, it's best to declare the winning combos as a constant: `const WINNING_COMBOS`